templates/etc/default/grub.j2

```diff
@@ -16,9 +16,12 @@ GRUB_TIMEOUT={{ grub_timeout }}
 GRUB_DISTRIBUTOR=$(lsb_release -i -s 2>/dev/null || echo {{ ansible_distribution }})
 GRUB_CMDLINE_LINUX_DEFAULT="{{ grub_console }}"
 GRUB_CMDLINE_LINUX="{{ grub_cmdline_linux | join(' ') }}"
-{% if grub_serial | d(False) %}
+{% if grub_serial_enabled == True %}
 GRUB_SERIAL_COMMAND="serial --speed={{ grub_serial.speed }} --unit={{ grub_serial.unit }} --word={{ grub_serial.word }} --parity={{ grub_serial.parity }} --stop={{ grub_serial.stop }}"
 {% endif %}
 
-# disable graphical terminal (grub-pc only)
+{% if grub_serial_enabled == True %}
```

Comment:
Just write `{% if grub_serial_enabled %}`, as this is shorter

Reply:
Why two consecutive equal `if`-statements?